### ISSUE TITLE
segmental_snr_mixed bug?

### ISSUE THREAD

**Issue by SiyuanWei** [microsoft/DNS-Challenge#94]
![image](https://user-images.githubusercontent.com/20530918/145747202-b121e364-f32c-474e-8f9c-df9c6771dcb8.png)
While the clean  audio and noise audio have already been normalized based on the RMS, I'm wondering that whether the noisescalar should be multiplied by rms_clean/ rms_noise..

**Comment by haozh7109** [comment]
I agree with @anteju, this is definitely a bug that needs to be fixed. Otherwise, the resulting generated noisy data will have incorrect SNR.

**Comment by YuXI-Chn** [comment]
I agree with @anteju, this is **definitely** a bug that needs to be fixed. Otherwise, the resulting generated noisy data will have incorrect SNR.
